pyinfra/operations/systemd.py

```diff
@@ -25,7 +25,7 @@ def service(
     service,
     running=True, restarted=False, reloaded=False,
     command=None, enabled=None, daemon_reload=False,
-    state=None, host=None,
+    state=None, host=None, user_mode=False,
```

Comment:
The `user_mode` argument here must live before `state` & `host`.

Sidenote: the documentation on this is _not_ explicit, I've added https://github.com/Fizzadar/pyinfra/issues/486 to track fixing that.

Reply:
Done!